samples/bluetooth/unicast_audio_server/boards/nrf52840dk_nrf52840.conf

```diff
@@ -0,0 +1,2 @@
+CONFIG_BT_CTLR_ADV_EXT=y
+CONFIG_BT_CTLR_ADV_PERIODIC=y
```

Comment:
Can you please also add similar file for the `nrf5340dk_nrf5340_cpuapp` board (`nrf5340dk_nrf5340_cpuapp.conf`)?

Reply:
> Can you please also add similar file for the `nrf5340dk_nrf5340_cpuapp` board (`nrf5340dk_nrf5340_cpuapp.conf`)?

Will do. However, I don't have a 5340dk, thus can't verify it.